telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/methods/SetUserEmojiStatus.java

```diff
@@ -43,7 +44,7 @@ public class SetUserEmojiStatus extends BotApiMethodBoolean {
      */
     @JsonProperty(USER_ID_FIELD)
     @NonNull
-    private Integer userId;
+    private Long userId;
```

Comment:
@rubenlagus you're absolutely correct. However, `Long` is used everywhere in this library except these 4 new methods

![image](https://github.com/user-attachments/assets/e3f60423-d398-4422-ba0a-af743b21ddf1)

I don't see any reason why the same entity `user id` could have different types. For consistency it would be great to use the same type for all user ids. And since `Long` is already used in most places, switching them all back to `Integer` might cause a compatibility problem, but switching `Integer` to `Long` in these 4 new methods won't break anything since the new version hasn't been released yet.

Reply:
For user ids it's specified in the docs:

![image](https://github.com/user-attachments/assets/f14e4950-95a9-4a58-a38f-a6ba6fefc8a9)